doc/style-guide-markdown.md

```diff
@@ -0,0 +1,42 @@
+Style Guide for Markdown
+========================
+
+This extends the [general programming](style-guide-programming.md) portion of the [Style Guide](style-guide.md).
+
+
+* Markdown Formatting
+    * Follow GitHub Flavored Markup.
+    * Paragraphs should be all on one long line.
+    * Avoid HTML markup.
+    * Prefer to use indentation instead of code fences.
```

Comment:
Indentation in lists is tricky because you have to double-indent the code.  Without the newlines before and after, some parsers (like GitHub) do not recognize the code block.  Here's an example of code inside of a list.  Note that I used "1." for both of the numbers in the comment when you view the raw version but the markdown-parsed version shows "2." for the second item.

1. list item

        code

1. another list item